source/client-side-operations-timeout/tests/client-side-encryption.yml

```diff
@@ -0,0 +1,79 @@
+runOn:
```

Comment:
This seems fine for now. In the future we'll be moving tests to their respective directories. At that point, this can be converted to the `.yml.template` format. Right?

Reply:
I assume you mean the `.yml.template` format in the FLE tests directory. IIUC, that format is used when testing all operations (e.g. if we wanted to assert that timeout is applied during the FLE section of each possible operation), but we're only testing `insertOne`. Does this need to be templated?

Reply:
Correct, I was referring to the files in the `source/client-side-encryption/etc/test-templates` directory. Those templated files were used to keep the key vault data, JSON schemas, and ciphertexts out of the files so they could be regenerated if the keys changed (e.g. if AWS / Azure / GCP credentials needed to be regenerated).